pages/projects/index.js

```diff
@@ -0,0 +1,5 @@
+function MembersPage() {
```

Comment:
The name should be `ProjectsPage`, right?
The export needs to be modified too.. :)

Reply:
You're right, didn't notice that :)